src/commons/locations.json

```diff
@@ -0,0 +1,18 @@
+{
+    "Seattle": "Seattle",
+    "San_Francisco": "San Francisco / Bay",
+    "Los_angeles": "Los Angeles",
```

Comment:
Capital A. Also I think RHS needs to be the snake case because that's what gets passed into template.